rules/gdi-aircraft.yaml

```diff
@@ -862,3 +862,45 @@ KODIAK:
 	WithVoxelBody:
 	WithMuzzleOverlay:
 	HitShape:
+
+Hellfire_beacon:
+	Inherits: ^AttackPlane
+	Inherits@FeatureTargettype: ^FeatureTargettypeFloraAir
```

Comment:
FeatureTargettypeFloraAir is used in the ability branch to make it untargetable.